ouster_pcap/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ target_include_directories(ouster_pcap PUBLIC
 CodeCoverageFunctionality(ouster_pcap)
 
 if(WIN32)
-  target_compile_options(ouster_pcap PRIVATE /wd4200)
+#  target_compile_options(ouster_pcap PRIVATE /wd4200)
```

Comment:
Not sure what to do about this one. Am I good to remove it? It's breaking builds on mingw which work otherwise.

Reply:
This warning isn't really removable: https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-levels-2-and-4-c4200?view=msvc-170
But we could silence in the code with a pragma instead if it still exists.

Reply:
For now I'll just remove the warning suppression